pages/index.js

```diff
@@ -1,9 +1,19 @@
-import Posts from '../src/Components/PostCard';
+import getPosts from '../src/Services/get-list';
+import PostGrid from '../src/Components/List.js/MapData';
```

Comment:
Your folder ends with .js ?! Mhhhh.... 🧐

Reply:
Oh yeah, that's right....